### PR TITLE
p0 completed

### PR DESCRIPTION
**Pull request by hrishikeshmane**

### CODE REVIEW

File: components/alert.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  max-width: 50rem;
+}
+.alert-icon {
+  margin-right: 1rem;
+}
+.alert-close {
+  padding: 0.1rem;
+  font-size: 1.8rem !important;
+  color: #4b4b4b;
+  background-color: #ffffff85;
```
Comment (ishitakeshawani) [inline components/alert.css:12]: You can use variables for color instead of directly giving value.

**Review by ishitakeshawani — COMMENTED** [review]
Leave a comment. Everything else looks good to me.

File: components/alert.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+.alert {
+  width: 100%;
+  max-width: 50rem;
+}
+.alert-icon {
+  margin-right: 1rem;
+}
+.alert-close {
+  padding: 0.1rem;
+  font-size: 1.8rem !important;
```
Comment (Aditi0505) [inline components/alert.css:10]: Suggestion: Try to increase specificity instead of using important.

File: components/button.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+.btn-floating {
+  width: 3.5rem;
+  height: 3.5rem;
+  color: var(--white);
+  background-color: rgb(236, 144, 52);
+}
+.btn-floating:hover {
+  background-color: rgb(255, 128, 0);
+  box-shadow: 0px 2px 1px -1px rgb(0 0 0 / 20%),
+    0px 1px 1px 0px rgb(0 0 0 / 20%), 0px 3px 10px 0px rgb(0 0 0 / 25%);
```
Comment (Aditi0505) [inline components/button.css]: Suggestion: Please use hex code for consistency.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions Rest looks good to me.

**Pull request merged**